bundle/src/main/java/com/adobe/acs/commons/rewriter/impl/StaticReferenceRewriteTransformerFactory.java

```diff
@@ -116,6 +120,8 @@ public void startElement(String namespaceURI, String localName, String qName, At
 
     private String[] staticHostPattern;
 
+    private Boolean replaceHost;
```

Comment:
Can you just use `boolean` here? It doesn't look like there is a code path where this is null, so the primitive would make more sense.